src/styles/page.scss

```diff
@@ -0,0 +1,26 @@
+.page {
+  font-family: 'Space Grotesk', sans-serif;
+  font-size: 16px;
```

Comment:
I see you use font-size in html tag with class 'page', so in previous places you can not write font-size: 16px;, because it's size for all tags